### PR TITLE
TRUNK-3343: Liquibase changeset to insert the new application level privileges for each corresponding api level privilege

### PR DESCRIPTION
**Pull request by puevigreven**

### CODE REVIEW

**Comment by djazayeri** [comment]
There are too many privileges here.

E.g. the first one doesn't make conceptual sense

    'Patient Overview - Get Concepts', 'Able to view the concept entries portlet on the patient overview tab'

**Comment by puevigreven** [comment]
@djazayeri 
In the latest commit i have kept following privileges.
Please guide me to refine the list. 

---------ADDING------------

- Patient Overview - Get Encounters
- Patient Overview - Get Encounter Types
- Patient Overview - Get Locations
- Patient Overview - Get Observations
- Patient Overview - Get Patient Identifiers
- Patient Overview - Get Orders
- Patient Overview - Get Visits
- Patient Overview - Get Providers

**Comment by puevigreven** [comment]
@djazayeri  thank you for your concern. :)

**Comment by dkayiwa** [comment]
@puevigreven did you see the rest of the patient dashboard privileges in https://github.com/openmrs/openmrs-core/blob/1.11.x/webapp/src/main/webapp/WEB-INF/view/patientDashboardForm.jsp

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -9056,4 +9056,28 @@
...
+		<comment>
+			Assigning the new Patient Overview privileges to roles with corresponding api level privileges
+		</comment>
+			<sql>
+				
+				INSERT INTO role_privilege (role, privilege)  
+				SELECT role, 'Patient Dashboard - View Visits Section' FROM role_privilege rp 
+				WHERE rp.privilege = 'View Visits'
+				and not exists (select role, privilege from role_privilege rp2 where rp2.privilege='Patient Dashboard - View Visits Section' and rp2.role=rp.role); 
+
```
Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: I thought on the ticket they mention the 'View Patient Programs' privilege too

**Comment by downeym** [comment]
This PR has been open for 94 days. What is its current status? 📅 🕚

**Comment by dkayiwa** [comment]
Closing because it has merge conflicts and misses a few more privileges as per my comment above.

**Pull request closed without merging**